src/hud.py

```diff
@@ -0,0 +1,42 @@
+import arcade
+
+from player import Player
+from playerhud import PlayerHud
+
+
+class Hud:
+
+    sprite: arcade.Sprite
+    player_hud_startx: int = 100
+    player_hud_starty: int = 575
+    hud_sprite_list: list = []
+    player_number_tracker: int = 0
```

Comment:
Nit: should move these assignments into `__init__`.  Sorry I missed this first time around.